### PR TITLE
Log: Add comments to entire logging server

### PR DESCRIPTION
**Pull request by JustNireon**

### CODE REVIEW

File: PatternPal/PatternPal.LoggingServer/Models/ProgSnap2.cs
Comment (ovda96) [inline PatternPal/PatternPal.LoggingServer/Models/ProgSnap2.cs:1]: For the entire thing: all are not xml-comments, right?

File: PatternPal/PatternPal.LoggingServer/Data/EventRepository.cs [hunk 1]
```diff
@@ -12,32 +12,55 @@ public EventRepository(ProgSnap2ContextClass context)
         {
             _context = context;
         }
-
+        /// <summary>
+        /// Returns list of all events
+        /// </summary>
+        /// <returns></returns>
```
Comment (ovda96) [inline PatternPal/PatternPal.LoggingServer/Data/EventRepository.cs:18]: For all subsequent comments: returns should include return type/description.

**Review by ovda96 — CHANGES_REQUESTED** [review]
I've added some things I feel are lacking in the current commenting; see specific files for more details!

**Review by WingToh — APPROVED** [review]
Good.

**Review by WingToh — APPROVED** [review]

**Pull request merged**